addons/project/__manifest__.py

```diff
@@ -218,6 +218,16 @@
             'project/static/src/project_sharing/search/favorite_menu/custom_favorite_item.xml',
             'project/static/src/project_sharing/**/*',
             'web/static/src/start.js',
+
+            # To use mail chatter in portal
+            'web/static/src/libs/pdfjs.js',
+            'bus/static/src/*.js',
+            'bus/static/src/services/**/*.js',
+            'bus/static/src/workers/websocket_worker.js',
+            'bus/static/src/workers/websocket_worker_utils.js',
+            ('include', 'portal.assets_chatter_helpers'),
+            'portal/static/src/chatter/core/**/*',
+            'project/static/src/chatter/**/*',
```

Comment:
This should have a bundle sub-folder (`portal` probably? or more specifically `project_portal` maybe?) and only import that rather than ** (on the bundle slot).

```suggestion
            'project/static/src/chatter/project_portal/*',
```

That way, if we have web specific override of chatter in project some day, there are no conflicts.

Reply:
You can put it inside `project_sharing` folder if you want. 🙂 

Since I guess the code inside chatter is just for project sharing (= portal view in edit mode), right? 🙂